### PR TITLE
textgroup_standardization

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated textgroup names to not include dates. Updated pseudo author.

### CODE REVIEW

File: data/stoa0104c/__cts__.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:stoa0104c">
-  <ti:groupname xml:lang="eng">Cyprianus Gallus active 5th century</ti:groupname>
+  <ti:groupname xml:lang="eng">Cyprianus Gallus</ti:groupname>
```
Comment (lcerrato) [inline data/stoa0104c/__cts__.xml:2]: attribute Latin

File: data/stoa0224b/__cts__.xml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:stoa0224b">
-  <ti:groupname xml:lang="eng">Paulinus Petricordiae active 459-472</ti:groupname>
+  <ti:groupname xml:lang="lat">Paulinus Petricordiae</ti:groupname>
```
Comment (lcerrato) [inline data/stoa0224b/__cts__.xml:2]: Latin attribute

File: data/stoa0292d/__cts__.xml [hunk 3]
```diff
@@ -1,3 +1,3 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:stoa0292d">
-  <ti:groupname xml:lang="eng">Victor Vitensis active 5th century</ti:groupname>
+  <ti:groupname xml:lang="eng">Victor Vitensis</ti:groupname>
```
Comment (lcerrato) [inline data/stoa0292d/__cts__.xml:2]: attribute Latin?

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/stoa0089a/__cts__.xml [hunk 4]
```diff
@@ -1,3 +1,3 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:stoa0089a">
-  <ti:groupname xml:lang="eng">Claudianus Mamertus -approximately 474</ti:groupname>
+  <ti:groupname xml:lang="eng">Claudianus Mamertus</ti:groupname>
```
[the remark this answers is not in the document]
Comment (AlisonBabeu) [reply data/stoa0089a/__cts__.xml]: Ok, I've done that and will make the other changes and resubmit.

**Pull request merged**